roads.mss

```diff
@@ -881,9 +1013,7 @@ tertiary is rendered from z10 and is not included in osm_planet_roads. */
           line-color: @secondary-fill;
         }
         [construction = 'tertiary'],
-        [construction = 'tertiary_link'] {
-          line-color: @tertiary-fill;
```

Comment:
Other white roads are not defined and overriden by this rule. Given that if constructions are unavailable I would keep it unchanged.